torch/_inductor/codegen/wrapper.py

```diff
@@ -1240,6 +1241,26 @@ def _generate(self, is_inference):
 
         return result.getvaluewithlinemap()
 
+    def _cleanup_autotune_example_tensors(self):
```

Comment:
I may want to rename this `_finalize_autotune_block` or something like that which indicates it should only be run once, right before autotuning.